server/sonar-web/src/main/js/apps/custom-measures/components/List.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { sortBy } from 'lodash';
+import DeleteButton from './DeleteButton';
+import EditButton from './EditButton';
+import { CustomMeasure } from '../../../app/types';
+import ActionsDropdown, {
+  ActionsDropdownDivider
+} from '../../../components/controls/ActionsDropdown';
+import { translate } from '../../../helpers/l10n';
+import Tooltip from '../../../components/controls/Tooltip';
+import { formatMeasure } from '../../../helpers/measures';
+import DateFormatter from '../../../components/intl/DateFormatter';
+
+interface Props {
+  measures: CustomMeasure[];
+  onDelete: (measureId: string) => Promise<void>;
+  onEdit: (data: { description: string; id: string; value: string }) => Promise<void>;
+}
+
+export default function List({ measures, onDelete, onEdit }: Props) {
+  return (
+    <div className="boxed-group boxed-group-inner" id="custom-measures-list">
+      {measures.length > 0 ? (
+        <table className="data zebra zebra-hover">
+          <thead>
+            <tr>
+              <th>{translate('custom_measures.metric')}</th>
+              <th>{translate('value')}</th>
+              <th>{translate('description')}</th>
+              <th>{translate('date')}</th>
+              <th />
+            </tr>
+          </thead>
+          <tbody>
+            {sortBy(measures, measure => measure.metric.name.toLowerCase()).map(measure => (
+              <tr data-metric={measure.metric.key} key={measure.id}>
+                <td className="nowrap">
+                  <div>
+                    <span className="js-custom-measure-metric-name">{measure.metric.name}</span>
+                    {measure.pending && (
+                      <Tooltip overlay={translate('custom_measures.pending_tooltip')}>
+                        <span className="js-custom-measure-pending badge badge-warning spacer-left">
+                          {translate('custom_measures.pending')}
```

Comment:
![image](https://user-images.githubusercontent.com/1725611/36100735-8de1d21c-1007-11e8-8bfe-11e1396914b9.png)


Is it still true with live measures ?

Reply:
I think so.